disasterinfosite/static/js/app.js

```diff
@@ -22,7 +22,7 @@ require("../img/locate-me.svg");
 require("../img/data-sources.pdf");
 
 // about page images
-require("../img/flowchart.png");
+require("../img/flowchart.jpg");
```

Comment:
Should there be a corresponding change in the static files?  I see a flowchart.png in `disasterinfosite/static/img` and no flowchart.jpg.

Reply:
I see this is fixed now!